okonomiyaki/platforms/__init__.py

```diff
@@ -1,9 +1,11 @@
 # flake8: noqa
 from .abi import PlatformABI, default_abi
 from .epd_platform import X86, X86_64, EPDPlatform, applies
-from .platform import Platform, OSKind, FamilyKind, NameKind
+from ._platform import Platform, OSKind, FamilyKind, NameKind
```

Comment:
`okonomiyaki.platforms.epd_platform` is not public either. It is generally understood that importing submodules is bad.

As for the reasoning, `platform` causes trouble because it is in the stdlib, and when I execute `_pep425_impl.py` as a subprocess, it fails importing on python 2.